facebook_scraper/extractors.py

```diff
@@ -203,6 +203,15 @@ def extract_time(self) -> PartialPost:
             except (KeyError, ValueError):
                 continue
 
+        try:
+            date = utils.parse_date(element_full_text=self.element.full_text)
+            if date:
+                return {
+                    'time': date
+                }
+        except:
```

Comment:
It's better to catch `Exception`:
```suggestion
        except Exception:
```
Or maybe let it fail since it should be caught and logged on the extract_post method.

Reply:
You're right, I'll let it fall. Should be caught later on.